<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">

   <item row="0" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Mine</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="lblMine">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>

   <item row="1" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Status</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="lblStatus">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>

   <item row="2" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="lblType">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Asset</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="lblAsset">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Quantity</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="lblQuantity">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   
   <item row="5" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Unit Price</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QDoubleSpinBox" name="spnUpdateUnitPrice">
     <property name="suffix">
      <string> RVN</string>
     </property>
     <property name="decimals">
      <number>8</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
     <property name="minimum">
      <double>0.00000000</double>
     </property>
     <property name="maximum">
      <double>21000000</double>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   
   <item row="6" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Total Price</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="lblTotalPrice">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>

   <item row="7" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>UTXO</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLabel" name="lblUTXO">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>

   <item row="8" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Destination</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="txtDestination">
     <property name="text">
      <string/>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>

   <item row="9" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Signed Partial</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QTextEdit" name="txtSigned">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   
   <item row="10" column="1">
    <widget class="QLabel" name="lblWarning">
     <property name="text">
      <string>Warning: A Signed Partial remains valid as long at the source UTXO is (details on GitHub.)&lt;br/&gt;Updating the price will only generate a new Signed Partial locally,&lt;br/&gt;this does not affect any previously created/shared Signed Partials for these same assets.</string>
     </property>
    </widget>
   </item>

   <item row="11" column="0">
    <widget class="QLabel" name="order">
     <property name="text">
      <string>Order Number</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QSpinBox" name="spnOrderNumber">
     <property name="value">
      <number>1</number>
     </property>
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>

   <item row="12" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
